models/setup_test.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+import (
+	"os"
+	"testing"
+
+	"github.com/gin-gonic/gin"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Why adding this file now?

Reply:
Because `TestMain` can declare only once per package [ref](https://medium.com/goingogo/why-use-testmain-for-testing-in-go-dafb52b406bc)

> Since function names need to be unique in a given package, you can only define the TestMain once for each package. If your package has multiple test files, choose a logic place for your single TestMain function.

Reply:
Okay, so basic reason indeed 😅